pkg/handler/v2/routes_v2.go

```diff
@@ -1161,16 +1165,16 @@ func (r Routing) createCluster() http.Handler {
 
 // swagger:route GET /api/v2/projects/{project_id}/clusters project listClustersV2
 //
-//     Lists clusters for the specified project. If query parameter `show_dm_count` is set to `true` then the endpoint will also return the number of machine deployments of each cluster.
+//	Lists clusters for the specified project. If query parameter `show_dm_count` is set to `true` then the endpoint will also return the number of machine deployments of each cluster.
 //
-//     Produces:
-//     - application/json
+//	Produces:
+//	- application/json
 //
-//     Responses:
-//       default: errorResponse
-//       200: ClusterList
-//       401: empty
-//       403: empty
+//	Responses:
+//	  default: errorResponse
+//	  200: ClusterList
+//	  401: empty
+//	  403: empty
 func (r Routing) listClusters() http.Handler {
```

Comment:
This whole file is getting formatted, can you please undo that to reduce the diff of this PR?

Reply:
cool, it was auto done by vscode and even goland, I couldn't figure out why.
But, now I removed then using text editor, only kept the change of `//swagger => // swagger` to keep it consistent